specifyweb/frontend/js_src/lib/components/SkeletonLoaders/AppResource.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import ContentLoader from 'react-content-loader';
+
+import { userPreferences } from '../Preferences/userPreferences';
+
+export function AppResourceSkeleton(): JSX.Element {
+  const [motionPref] = userPreferences.use('general', 'ui', 'reduceMotion');
+  return (
+    <div className="h-full w-full">
+      <ContentLoader
+        animate={motionPref !== 'reduce'}
+        backgroundColor="#333"
```

Comment:
also, whatever colors you settle on, refactor the code in some way so that you don't have to specify these colors for every skeleton.

i.e, either assign them to a variable, or create a wrapper component around <ContentLoader> that would also provide the value for animate and speed props

if you were on the jscpd branch, I'm sure it will complain about all this code duplication